ui/plugins/pluginpanel.html

```diff
@@ -107,6 +107,16 @@
               <span class="fa fa-bar-chart-o"></span>
             </a>
           </li>
+          <li>
+            <a title="Back fill this till last seen job"
```

Comment:
Just a drive-by comment, it would be a bit clearer to use the words 'Backfill' and 'until' here.